appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/PayaraMicro.java

```diff
@@ -265,6 +275,30 @@ public void setUserLogFile(String fileName) {
     }
     
     /**
+     * Set user defined file directory for the access log
+     *
+     * @param filePath
+     */
+    public void setAccessLogDir(String filePath) {
+        if (!filePath.endsWith("/")) {
+
```

Comment:
In any case, if there is an unexpected value in the argument, Payara Micro should log this as an error. I could not see anything in the logs to give me a hint on what I was doing wrong (that I did not add "/" at the end of the path) and I had to look into the source code.